roles/ceph-osd/tasks/scenarios/lvm.yml

```diff
@@ -1,164 +1,17 @@
-  command: "ceph-volume --cluster {{ cluster }} lvm create --filestore --data {{ item.data }} --journal {{ item.journal }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - item.journal_vg is not defined
-    - item.data_vg is not defined
-    - osd_objectstore == 'filestore'
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create filestore osds with dedicated lv journals and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --filestore --data {{ item.data }} --journal {{item.journal_vg }}/{{ item.journal }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - item.journal_vg is defined
-    - item.data_vg is not defined
-    - osd_objectstore == 'filestore'
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with db and wal and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }} --block.wal {{ item.wal_vg }}/{{ item.wal }} --block.db {{ item.db_vg }}/{{ item.db }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - osd_objectstore == 'bluestore'
-    - item.data_vg is not defined
-    - item.wal is defined
-    - item.wal_vg is defined
-    - item.db is defined
-    - item.db_vg is defined
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with db only and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }} --block.db {{ item.db_vg }}/{{ item.db }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - osd_objectstore == 'bluestore'
-    - item.data_vg is not defined
-    - item.wal is not defined
-    - item.wal_vg is not defined
-    - item.db is defined
-    - item.db_vg is defined
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with wal only and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }} --block.wal {{ item.wal_vg }}/{{ item.wal }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - osd_objectstore == 'bluestore'
-    - item.wal is defined
-    - item.data_vg is not defined
-    - item.wal_vg is defined
-    - item.db is not defined
-    - item.db_vg is not defined
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with just a data device and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }}"
+- name: use ceph-volume to create osds
```

Comment:
I think it'll be nice for debugging purpose to still expose some of the variables consumed by the task in the "name".

Reply:
It does show nicely in the output from the task, see ``item`` below.

```
TASK [ceph-osd : use ceph-volume to create osds] *******************************
task path: /home/jenkins-build/build/workspace/ceph-ansible-scenario/roles/ceph-osd/tasks/scenarios/lvm.yml:3
Friday 01 December 2017  14:52:17 +0000 (0:00:00.057)       0:08:31.623 ******* 
changed: [osd0] => (item={u'data': u'data-lv1', u'data_vg': u'test_group'}) => {
    "changed": true, 
    "cmd": [
        "ceph-volume", 
        "--cluster", 
        "test", 
        "lvm", 
        "create", 
        "--bluestore", 
        "--data", 
        "test_group/data-lv1"
    ], 
    "delta": "0:00:23.018566", 
    "end": "2017-12-01 14:52:42.832223", 
    "failed": false, 
    "item": {
        "data": "data-lv1", 
        "data_vg": "test_group"
    }, 
    "rc": 0, 
    "start": "2017-12-01 14:52:19.813657"
}
```
I'm unsure if I can modify the ``name`` of the task from the module.

Reply:
It's showing up nicely here because we use `-vv` I'm thinking more of someone reporting a bug that ran ansible without any verbose options.

Reply:
@leseb I tried something in https://github.com/ceph/ceph-ansible/pull/2227/commits/f876a8173d8fbe30d8c5ab2534c5e76215858a42 but it doesn't seem to be working. It doesn't render ``{{ item | to_nice_yaml }}`` there at all. Any suggestions on how to accomplish this?